doc/examples/planning_adapters/planning_adapters_tutorial.rst

```diff
@@ -200,8 +200,7 @@ This section has insights as to when to use which planner and how using certain
 
 For more information on each of these motion planners, refer to their individual tutorial pages
 :doc:`OMPL </doc/examples/ompl_interface/ompl_interface_tutorial>`,
-:doc:`CHOMP </doc/examples/chomp_planner/chomp_planner_tutorial>` and
-:doc:`STOMP </doc/examples/stomp_planner/stomp_planner_tutorial>`.
+:doc:`CHOMP </doc/examples/chomp_planner/chomp_planner_tutorial>`.
```

Comment:
```suggestion
:doc:`CHOMP </doc/examples/chomp_planner/chomp_planner_tutorial>`, and
:doc:`STOMP </doc/how_to_guides/stomp_planner/stomp_planner>`.
```
STOMP was moved out of examples with #682, since examples basically was a catch-all phrase for ROS 1 content.